GooglePlayInstant/Editor/PlayInstantAssetBundleUrlVerifierWindow.cs

```diff
@@ -0,0 +1,135 @@
+        /// Creates a dialog box that details the success or failure of an AssetBundle retrieval from a given assetBundleUrl.
+        /// </summary>
+        public static void ShowWindow()
+        {
+            if (string.IsNullOrEmpty(QuickDeployConfig.Config.assetBundleUrl))
+            {
+                Debug.LogError("Problem with verifying AssetBundle: AssetBundle URL cannot be empty.");
+            }
+            else
+            {
+                // Set AssetBundle url in a private variable so that information displayed in window is consistent with
+                // the url that this was called on. 
+                _assetBundleUrl = QuickDeployConfig.Config.assetBundleUrl;
+                UpdateAssetBundleVerificationInfoWindow();
+                GetWindow(typeof(PlayInstantAssetBundleUrlVerifierWindow), true, "Play Instant AssetBundle Verify");
+            }
+        }
+
+        //TODO: Support Unity 5.6.0+
+        private static void UpdateAssetBundleVerificationInfoWindow()
+        {
+            var www = UnityWebRequestAssetBundle.GetAssetBundle(_assetBundleUrl);
+            www.SendWebRequest();
+            while (!www.isDone)
+            {
+                //TODO: implement loading bar 
+            }
+
+            var bundle = DownloadHandlerAssetBundle.GetContent(www);
+
+            _responseCode = www.responseCode;
+
+            if (www.isNetworkError || www.isHttpError)
+            {
+                _assetBundleDownloadIsSuccessful = false;
+                _errorDescription = www.error;
+                Debug.LogErrorFormat("Problem retrieving AssetBundle from {0}: {1}", _assetBundleUrl,
+                    _errorDescription);
+            }
+            else if (bundle == null)
+            {
+                _assetBundleDownloadIsSuccessful = false;
+                _errorDescription = "Failed to decompress data for the AssetBundle.";
+                // No need to log since debugging information in this case is automatically logged by Unity.
+            }
+            else
+            {
+                _assetBundleDownloadIsSuccessful = true;
+                _numOfMegabytes = ConvertBytesToMegabytes(www.downloadedBytes);
+
+                // Free memory used by the AssetBundle since it will not be in use by the Editor. Set to true to destory
+                // all objects that were loaded from this bundle.
+                bundle.Unload(true);
+            }
+        }
+
+        private static double ConvertBytesToMegabytes(ulong bytes)
+        {
+            return bytes / 1024f / 1024f;
+        }
+
+        private void OnGUI()
+        {
+            EditorGUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField("AssetBundle Download Status:", GUILayout.MinWidth(FieldMinWidth));
+            EditorGUILayout.LabelField((_assetBundleDownloadIsSuccessful ? "SUCCESS" : "FAILED"));
+            EditorGUILayout.EndHorizontal();
+            EditorGUILayout.Space();
```

Comment:
Each of these 5 blocks looks almost identical. Can you extract a method with 2 params for the title and value to use in each case?